SCons/Tool/ninja/NinjaState.py

```diff
@@ -209,7 +232,7 @@ def __init__(self, env, ninja_file, ninja_syntax):
                 "restat": 1,
             },
             "TEMPLATE": {
-                "command": f"{sys.executable} {pathlib.Path(__file__).parent / 'ninja_daemon_build.py'} {scons_daemon_port} {get_path(env.get('NINJA_DIR'))} $out",
+                "command": f"{sys.executable} {pathlib.Path(__file__).parent / 'ninja_daemon_build.py'} $PORT {env.get('NINJA_DIR').abspath} $out",
```

Comment:
would it be worth changing `{env.get('NINJA_DIR').abspath}` to $ABS_NINJA_DIR and define it up in self.variables?

Reply:
yeah I also did several others to clean it up a bit